### ISSUE TITLE
request: investigate adding snapraid to ucore

### ISSUE THREAD

**Issue by bsherman** [ublue-os/ucore#25]
I would like snapraid and it was brought up in discord chat so this would be nice to add.

https://www.snapraid.it/

At one time there were epel/fedora builds in pagure, but no longer. Investigate how to add.

**Comment by bsherman** [comment]
I did some testing on building a snapraid RPM.  Testing is successful, but I want to see how the snapraid author would like to handle this.

Currently there is a launchpad (ubuntu) PPA for snapraid, so we could build a COPR as a similar solution. 

I've submitted a PR to the author to see if there is interest in owning the spec file (or even the RPM build) before I proceed further.

https://github.com/amadvance/snapraid/pull/14
